platform/linux-generic/odp_queue_scalable.c

```diff
@@ -844,6 +846,11 @@ static int queue_deq_multi(odp_queue_t handle, odp_event_t ev[], int num)
 		num = QUEUE_MULTI_MAX;
 
 	queue = qentry_from_ext(handle);
+
+	if (odp_global_rw->inline_timers &&
+	    odp_atomic_load_u64(&queue->s.timers))
+		timer_run();
```

Comment:
check for inline timers is not needed because it's done inside timer_run():
```
static inline unsigned timer_run(void)
{
	return inline_timers ? _timer_run() : 0;
}
```

Reply:
The `odp_global_rw->inline_timers` check is here to avoid unnecessary atomic load in case of inline timer are not used. The compiler should be able to optimise away the second check from the inlined `timer_run()` function.